gbus/invocation.go

```diff
@@ -51,6 +52,13 @@ func (dfi *defaultInvocationContext) Reply(ctx context.Context, replyMessage *Bu
 	return err
 }
 
+func (dfi *defaultInvocationContext) ReturnDeadToQueue(ctx context.Context, publishing *amqp.Publishing) error {
```

Comment:
Since I suggest we remove the ReturnDeadToQueue from the Messaging interface we should remove it from the invocation as well.
I think it makes sense as well as I do not think it makes much sense to automatically return a message to the queue from a handler but only after some manual introspection and action and in that case, the calling code would interact with the gbus.Bus interface (where the ReturnDeadToQueue would be available) and not with the Invocation interface